ldms/src/ldmsd/ldmsd_request.c

```diff
@@ -1825,16 +1831,22 @@ static int prdcr_start_regex_handler(ldmsd_req_ctxt_t reqc)
 		goto send_reply;
 	}
 
-	interval_str = ldmsd_req_attr_str_value_get_by_id(reqc, LDMSD_ATTR_INTERVAL);
-
+	interval_str = ldmsd_req_attr_str_value_get_by_id(reqc, LDMSD_ATTR_RECONNECT);
+	if (!interval_str) {
+		interval_str = ldmsd_req_attr_str_value_get_by_id(reqc,
+							LDMSD_ATTR_INTERVAL);
+		ovis_log(config_log, OVIS_LWARN, "The 'interval' in "
```

Comment:
I don't think we need these messages in the server.

Reply:
The log message in the server is for the case that the 'interval' attribute is used in configuration files.